### DISCUSSION TITLE
<title>site not update</title>

### DISCUSSION THREAD

**Discussion by nikolay-mansas** [community/community#44550] [Codespaces]
### Select Topic Area

Bug

### Body

I upload site code, visit site and site not upgrade

**Comment by idadwind1** [comment]
[Is this your website?](https://github.com/nikolay-mansas/nikolay-mansas.github.io)
Try creating an index.html

**Reply by idadwind1** [reply]
seems like your problem is solved
remember to set this discussion to solved

**Comment by nikolay-mansas** [comment]
I recreate github and reload all file, site is working! nikolay-mansas.github.io is working
